docs/developer/architecture/centralized-cli.rst

```diff
@@ -0,0 +1,57 @@
+Centralized CLI
+===============
+
+Context
+-------
+
+MetalK8s comes with a set of services to operate and monitor the K8s cluster.
+All operations that need to be performed by the Platform Administrator could be
+categorized as follow:
+- Cluster Resources Administration (Nodes, Volumes, Deployments, ...)
+- Cluster Administration (Install, Upgrade, Downgrade, Backup, Restore, ...)
+- Solution Administration (CRUD Environment, Import/Remove Solution, ...)
+- Cluster Service Administration (Configure Dex, Prometheus, Alert Manager, ...)
+
+K8s provides the kubctl CLI, which is distributed and enables all kind of
+interaction with all resources available in etcd DB but its usage often requires
```

Comment:
Not sure you want to talk about etcd here as kubectl basically just interact with apiserver he don't really care about etcd

Reply:
apiserver is just a giant wrapper in front of the etcd cluster anyway

Reply:
For the moment yes but maybe one day they will use another backend :)

Reply:
yes agree @TeddyAndrieux will change to apiserver

Reply:
done